.github/workflows/integration_tests.yml

```diff
@@ -88,8 +106,19 @@ jobs:
           node-version: 20
           cache: "npm"
           cache-dependency-path: front_end/package-lock.json
-      - name: "Install node_modules and build the frontend"
-        run: cd front_end && npm ci && npm run build
+      - name: Cache node_modules
+        id: cache-node-modules
+        uses: actions/cache@v4
+        with:
+          path: front_end/node_modules
+          key: node-modules-${{ runner.os }}-${{ hashFiles('front_end/package-lock.json') }}
+          restore-keys: |
+            node-modules-${{ runner.os }}-
+      - name: Install node_modules
+        if: steps.cache-node-modules.outputs.cache-hit != 'true'
+        run: cd front_end && npm ci
+      - name: Build the frontend
+        run: cd front_end && GENERATE_SOURCE_MAPS=false npm run build
```

Comment:
Any reason why we don't build with source maps? I am thinking that if for some reason the tests fail, wouldbe nice to have the source maps?